gradio/components.py

```diff
@@ -4046,6 +4046,22 @@ def style(self, container: Optional[bool] = None):
         )
 
 
+class AltairPlot:
```

Comment:
I expect the number of methods in `AltairPlot` to increase as we add more plots and find more common patterns that can be abstracted out.

For the time being I don't want to invest too much time in coming up with a fancy abstraction as the plotting code will be edited a lot over the coming months.